server/core_leaderboard.go

```diff
@@ -339,6 +340,26 @@ func LeaderboardRecordsList(ctx context.Context, logger *zap.Logger, db *sql.DB,
 		_ = rows.Close()
 	}
 
+	// Sort owner records according to leaderboard ordering
+	var sortFn func(i, j int) bool
+	if leaderboard.SortOrder == LeaderboardSortOrderAscending {
+		sortFn = func(i, j int) bool {
```

Comment:
Cached records sorting fallbacks to comparing user ids in case scores and subscores are equal:
https://github.com/heroiclabs/nakama/blob/master/server/leaderboard_rank_cache.go#L50-L63

Just wondering if we want to replicate this logic here too?

Reply:
Yep that's a fair point, I'll do the same to keep consistency 👍